### DISCUSSION TITLE
What is the fastest way to estimate many models with different initial conditions and iter.max = 0?

### DISCUSSION THREAD

**Discussion by billdenney** [nlmixr2/nlmixr2est#462] [Q&A]
In the near future, I'd like to implement sampling-importance-resampling (SIR).  The way that this is done within PsN is that it creates thousands of $EST steps within a single NONMEM control stream.  Is something similar feasible within `nlmixr2est::nlmixr2()`?

I think that the answer is no, and so the fastest way that I can think to do it is many calls to `nlmixr2`, but I would guess that there is some step deeper in the estimation process where the model file has been processed and possibly just before a C++ call where we may be able to provide new initial estimates.  I would also guess that part is not exported.

**Comment by mattfidler** [comment]
This needs to be probably discussed in more depth (probably at a developer meeting).  There are many way to speed this up, though I need to understand the problem a bit better to be able to help here.
